### ISSUE TITLE
Generic "Click Here" or "Read More" in link text

### ISSUE THREAD

**Issue by lfrancis** [forumone/peacecorps-site#402]
Don't use generic text like "Click Here" or "Read More" in link text, because it says nothing about the link target when read out by a screen reader. WCAG 2.0 AAA F84 

NOTE: We typically resolve this by  incorporating the header text in link text, so in this example at
https://stage-donate.peacecorps.gov/donate/projects-funds/#issue?subSection=issue-1&project=collapsible-project-1745 we'd change _read more_ to _read more about the Drinking Water and Sanitation Fund_.

 Screen readers often tab from one link to the next. Tabbing between links labelled "click here" sounds like "Click here, tab, click here, tab, click here". Change the link text to make sense when read without the surrounding text. 
Additional link text added with the TITLE attribute doesn't work in:
•NVDA 2014.1 with IE11 (the link title isn't read when tabbing)
•JAWS 14 with IE11 or Firefox 29 (the link title isn't read when tabbing)
•WindowEyes 8.4 with IE11 (the link title isn't read when tabbing)

 Link text: "read more"
https://stage-donate.peacecorps.gov/donate/projects-funds/  at lines: 1453 1492 1531 1570 1609 1648 1685 1724 1763 1802 1841 1878 1917 2020 2088 2255 2323 2424 2855 3088 3387 3455 3556 3690 4092 4189 4288 4392 4496 4593 4690 4794 4898 4995 5092 5194 5298 5395 5492 5596 5700 5797 5901 5998 6102 6210 6312 6416 6513 6610 6712 6816 6913 7017 7114 7223 7322 7419 7521 7618 7715 7819 7916 8013 8117 8221 8318 8422 8519 8616 8713 8810 8907 9011 9115 9212 9309 9406 9503 9600 9697 9794 9891 9988 10085 10182 10286 10383 10480 10577 10686 10783 10885 10982 11079 11183 11280 11377 11474 11571 11675 11772 11876 11980 12082 12179 12283 12380 12477
